TheForceEngine/TFE_Settings/dfLogics.cpp

```diff
@@ -0,0 +1,301 @@
+#include <cstring>
+#include <vector>
+#include <TFE_Settings/dfLogics.h>
+#include <TFE_Settings/logicTables.h>
+#include <TFE_System/cJSON.h>
+#include <TFE_System/system.h>
+#include "logicTables.h"
+
+namespace TFE_Settings
+{
+	///////////////////////////
+	/// Forward declarations 
+	///////////////////////////
+	bool tryAssignProperty(cJSON* data, CustomActorLogic& customLogic);
+
+	
+	void parseLogicData(char* data, std::vector<CustomActorLogic> &actorLogics)
```

Comment:
Super nitpicky, but in the code I usually pair the * or & with the type for pointers/references, aka:
char* data, std::vector<...>& actorLogics.

Reply:
Sure, will amend this